arelle/RenderingResolution.py

```diff
@@ -180,8 +181,10 @@ def jsonStrctMdlEncoder(obj, indent="\n"):
             # print(str(o))
             return o
         raise TypeError("Type {} is not supported for json output".format(type(obj).__name__))
-    with io.open(r"/Users/hermf/temp/test.json", 'wt') as fh:
-        json.dump(strctMdlTable, fh, ensure_ascii=False, indent=2, default=jsonStrctMdlEncoder)
+
+    if TRACE_TABLE_STRUCTURE:
+        with io.open(r"/Users/hermf/temp/test.json", 'wt') as fh:
+            json.dump(strctMdlTable, fh, ensure_ascii=False, indent=2, default=jsonStrctMdlEncoder)
```

Comment:
@hefischer I putted this code in an if, to make it work on other computer. If this PR is merge, you will need to change the TRACE_TABLE_STRUCTURE variable to have the same behavior as before